specification/visualstudio/resource-manager/Microsoft.VisualStudio/2014-04-01-preview/examples/CreateProjectResource.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "parameters": {
+    "body": {
+        "id": "/subscriptions/0de7f055-dbea-498d-8e9e-da287eedca90/resourceGroups/VS-Example-Group/providers/Microsoft.VisualStudio/account/ExampleAccount/project/ExampleProject",
+        "name": "ExampleProject",
+        "type": "Microsoft.VisualStudio/account/project",
+        "location": "Central US",
+        "tags": {},
+        "properties": {
+          "ProcessTemplateId": "6B724908-EF14-45CF-84F8-768B5384DA45",
```

Comment:
I made a comment about this [here](https://github.com/Azure/azure-rest-api-specs/pull/1498/files#diff-3f2a2429b806e0c507ad43a0e8aefc09R1084). I think that should work, but you could quickly verify that by generating C# and viewing the definition for `ProjectResource`